ksqldb-api-client/pom.xml

```diff
@@ -54,6 +54,13 @@
 
         <!-- Required for running tests -->
 
+        <dependency>
+            <groupId>io.confluent.ksql</groupId>
```

Comment:
I'm not sure why this additional dependency is needed beyond the one below (with type `test-jar`). If someone with more Maven knowledge can explain why that'd be great 😄